docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
is this file needed?

Reply:
I was generate this one with `sphinx-quickstart`